{
  "Header": {
    "Original": "CVE-2018-16427 (Various out of bounds reads when handling responses in OpenSC before 0 ...)",
    "Line": 280463,
    "ID": "CVE-2018-16427",
    "Description": "(Various out of bounds reads when handling responses in OpenSC before 0 ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-1916-1}",
      "Line": 280464,
      "Type": "xref",
      "Bugs": [
        "DLA-1916-1"
      ]
    },
    {
      "Original": "- opensc 0.19.0~rc1-1 (low; bug #909444)",
      "Line": 280465,
      "Type": "package",
      "Package": "opensc",
      "Kind": "fixed",
      "Version": "0.19.0~rc1-1",
      "Severity": "low",
      "BugNo": 909444
    },
    {
      "Original": "[stretch] - opensc 0.16.0-3+deb9u1",
      "Line": 280466,
      "Type": "package",
      "Release": "stretch",
      "Package": "opensc",
      "Kind": "fixed",
      "Version": "0.16.0-3+deb9u1"
    },
    {
      "Original": "NOTE: https://github.com/OpenSC/OpenSC/pull/1447/commits/8fe377e93b4b56060e5bbfb6f3142ceaeca744fa",
      "Line": 280467,
      "Type": "NOTE",
      "Description": "https://github.com/OpenSC/OpenSC/pull/1447/commits/8fe377e93b4b56060e5bbfb6f3142ceaeca744fa"
    },
    {
      "Original": "NOTE: https://www.x41-dsec.de/lab/advisories/x41-2018-002-OpenSC/",
      "Line": 280468,
      "Type": "NOTE",
      "Description": "https://www.x41-dsec.de/lab/advisories/x41-2018-002-OpenSC/"
    }
  ]
}